libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/idp/IDPAccountPickerActivity.java

```diff
@@ -153,7 +153,7 @@ protected void accountSelected(UserAccount account) {
          */
         if (account != null) {
             proceedWithIDPAuthFlow(account);
-        } else {
+        } else if (!SalesforceSDKManager.getInstance().isIDPAppLoginFlowActive()) {
```

Comment:
This one ensures we don't kick off a new login flow when one is active currently. This can occur sometimes because once `LoginActivity` goes away, control comes back on `onResume()` at which point we don't want to kick off another login flow.